File: app/src/main/java/workshop/akbolatss/tagsnews/screen/favorites/FavoritesView.java
package workshop.akbolatss.tagsnews.screen.favorites;

import java.util.List;

import me.toptas.rssconverter.RssItem;
import workshop.akbolatss.tagsnews.base.BaseView;

public interface FavoritesView extends BaseView {

    public void OnLoadFavorites(List<RssItem> rssItems);

}

File: app/src/main/java/workshop/akbolatss/tagsnews/screen/favorites/FavoritesView.kt
package workshop.akbolatss.tagsnews.screen.favorites

import workshop.akbolatss.tagsnews.base.BaseView
import workshop.akbolatss.tagsnews.model.dao.RssFeedItem

/**
 * MVP View for #FavoritesView
 * @see FavoritesActivity
 */
interface FavoritesView : BaseView {

    fun onLoadFavorites(rssItems: List<RssFeedItem>)

    fun onNoContent(isEmpty: Boolean)

}
